lib/style_parser/namespacer/index.js

```diff
@@ -35,5 +35,49 @@ function namespace(styles, prefix, postfix, cb) {
   ], cb);
 }
 
+function stripNamespace(styles, prefix, postfix, cb) {
+  async.waterfall([
+    // strip styles
+    function(cb) {
+      if (prefix) {
+        async.whilst(function() {
+          return (styles.indexOf(prefix) > -1);
+        }, function(cb) {
+          prefixer.strip(styles, prefix, function(err, reverted) {
+            styles = reverted;
+            cb();
+          });
+        }, function(err) {
+          if (err) return cb(err);
+          return cb(null, styles);
+        });
+      } else {
+        cb(null, styles);
+      }
+    },
+    // strip styles
+    function(prefixedStyles, cb) {
+      if (postfix) {
+        async.whilst(function() {
```

Comment:
This could cause it to crash as well if the postfix is part of a class name that's not at the beginning.